### ISSUE TITLE
AttributeError: 'str' object has no attribute 'sem_extract_base' with 2.4.0

### ISSUE THREAD

**Issue by Apfelwurm** [dvershinin/lastversion#79]
Hey, since the new 2.4.0 i get an error when calling

```
lastversion https://github.com/arendst/Tasmota --format tag
```

```
Traceback (most recent call last):
  File "/root/.platformio/penv/bin/lastversion", line 8, in <module>
    sys.exit(main())
  File "/root/.platformio/penv/lib/python3.10/site-packages/lastversion/lastversion.py", line 593, in main
    res = res.sem_extract_base(args.sem)
AttributeError: 'str' object has no attribute 'sem_extract_base'
```

with a force installation of 2.3.0 everything works like it's intended.

**Comment by dvershinin** [comment]
Thank you for the bug report. Added more tests, and fixed in 2.4.1

**Comment by Apfelwurm** [comment]
tested, everything works again :)
